README.md

```diff
@@ -114,6 +114,7 @@ embedded.kafka.topicsToCreate=some_topic
 * embedded.mariadb.dockerImage `(default is set to 'mariadb:10.3.2')`
   * You can pick wanted version on [dockerhub](https://hub.docker.com/r/library/mariadb/tags/)
 * embedded.mariadb.waitTimeoutInSeconds `(default is 60 seconds)`
+* embedded.mariadb.initScriptPath `(default is null)`
```

Comment:
guess worth adding more description for this property, also would be nice if you will add spring property description to ```additional-spring-configuration-metadata.json```